src/autoscaler/metricsgateway/helpers/ws_helper.go

```diff
@@ -72,6 +72,15 @@ func (wh *wshelper) SetupConn() error {
 				return err
 			}
 		} else {
+			go func() {
```

Comment:
why you do this?

Reply:
@boyang9527  control message handle processes are in "advanceFrame" when calling "readMessage"